main.go

```diff
@@ -52,7 +53,12 @@ func main() {
 
 	switch {
 	case token != "":
-		ses, err = session.New(token)
+		if os.Getenv("BOT") != "" {
```

Comment:
Is there a reason why `TOKEN="Bot abc"` wouldn't work and would need `BOT="*"` instead?

Reply:
None at all it seems, I can revert this part if you'd like.

Reply:
That would be nice.

Reply:
There you go.